<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTablasAuxiliares</class>
 <widget class="QDialog" name="frmTablasAuxiliares">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gnuOptics &gt; Tablas auxiliares</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tablas Auxiliares</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabTipos">
        <attribute name="title">
         <string>Tipos de art√≠culos</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="tblTipos">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tipos</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionTiposNuevo">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/star.png</normaloff>:/gnuoptics/star.png</iconset>
   </property>
   <property name="text">
    <string>Tipo nuevo</string>
   </property>
  </action>
  <action name="actionTiposBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar tipo</string>
   </property>
  </action>
  <action name="actionTiposModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar tipo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
